components/CookieControl.js

```diff
@@ -0,0 +1,76 @@
+export default function loadCookieControl(apiKey) {
+  var config = {
+    apiKey: apiKey,
+    product: 'community',
+    optionalCookies: [
+      {
+        name: 'analytics',
+        label: 'Analytics',
+        description: 'Analytical cookies help us to improve our website by collecting and reporting information on its usage.',
+        cookies: [],
+        onAccept: function() {
+          // Enable Google Analytics
+          window.dataLayer = window.dataLayer || [];
+          function gtag(){dataLayer.push(arguments);}
+          gtag('js', new Date());
+          gtag('config', '${gtag.GA_TRACKING_ID}', {
+            page_path: window.location.pathname,
+            cookie_flags: 'SameSite=None;Secure'
+          });
+
+          // Enable Google Tag Manager
+          (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+          new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+          j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+          'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+          })(window,document,'script','dataLayer','GTM-WF28PJS');
+        },
+        onRevoke: function(){
+          // Disable Google Analytics
+          window['ga-disable-${gtag.GA_TRACKING_ID}'] = true;
+
+          // Disable Google Tag Manager
+          var dataLayer = window.dataLayer || [];
+          dataLayer.push({
+            'event': 'gtm.disable',
+            'gtm.start': new Date().getTime()
+          });
+        }
+      }
+    ],
+    position: 'RIGHT',
+    theme: 'DARK'
+  };
+
+  var head = document.getElementsByTagName('head')[0];
+
+  // Google Tag Manager script
+  var gtmScript = document.createElement('script');
+  gtmScript.src = 'https://www.googletagmanager.com/gtm.js?id=GTM-WF28PJS';
+  gtmScript.async = true;
+  head.appendChild(gtmScript);
+
+  // Google Analytics script
+  var gaScript = document.createElement('script');
+  gaScript.src = 'https://www.googletagmanager.com/gtag/js?id=GA_TRACKING_ID';
+  gaScript.async = true;
+  head.appendChild(gaScript);
+
+  // Inline script for initializing Google Analytics
+  var gaInlineScript = document.createElement('script');
+  gaInlineScript.innerHTML = `
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+  `;
+  head.appendChild(gaInlineScript);
```

Comment:
Just looking at this again @HarleyBurrow and looking at the docs for Civic and there's a couple of things to flag here:

* I'm not sure we should be adding these scripts outside of the Cookie Control config, otherwise we may be tracking people regardless of their preferences 😅 
* Do we actually need these, or are they just duplicating what's already in the `onAccept` function in the config?

Finally, I'm wondering about lines 68-75. In general I think it's better to avoid adding HTML elements to the page using JS unless we really need to. Can you try to add the Cookie Control script directly in `pages/_app.js` please?